.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org
```

Comment:
We don't want user settings to decide? Was there a problem hitting internal npm registries?

Reply:
We want the registry to be npm for this package.